server/src/weaverbird/backends/pypika_translator/translators/base.py

```diff
@@ -683,10 +718,33 @@ def _get_single_condition_criterion(
                     return column_field.isnotnull()
 
             case DateBoundCondition():
-                if condition.operator == "until":
-                    return column_field <= condition.value
-                elif condition.operator == "from":
-                    return column_field >= condition.value
+
+                if isinstance(condition.value, (RelativeDate, datetime, str)):
+                    if isinstance(condition.value, RelativeDate):
+                        value_str_time = (
+                            evaluate_relative_date(condition.value)
+                            .astimezone()
```

Comment:
This converts the TZ to the system time, and forces the TZ on naive objects. I thought we agreed on UTC ?

Reply:
I may not getting well these TZ manipulation but, adding as argument `timezone.utc` to astimezone, force me to change fixtures tests, otherwise tests will not work anymore and we should not change fixture tests !

That's why i remove that parameter !

These are tests failing !
```
tests/backends/sql_translator_integration_tests/test_sql_postgres_pypika_translator_steps.py::test_sql_translator_pipeline[/filter_dates_from_until_pypika]
FAILED tests/backends/sql_translator_integration_tests/test_sql_postgres_pypika_translator_steps.py::test_sql_translator_pipeline[/filter_relative_dates_pypika]
FAILED tests/backends/sql_translator_integration_tests/test_sql_postgres_pypika_translator_steps.py::test_sql_translator_pipeline[/ifthenelse_dates_conditions_pypika]
```

Can you be more precise ?